aiohttp/helpers.py

```diff
@@ -676,7 +676,8 @@ def __call__(self) -> None:
 
 
 class BaseTimerContext(ContextManager["BaseTimerContext"]):
-    pass
+    def check_timeout(self) -> None:
```

Comment:
"check" implies that it'll return a boolean. Let's be explicit about the side effects:
```suggestion
    def assert_timeout_exceeded(self) -> None:
```

Reply:
Though you might come up with a better name.